lib/cogs/Lobbies.py

```diff
@@ -350,6 +350,19 @@ def __init__(self, LobbyPrompt) -> None:
     async def on_submit(self, interaction: discord.Interaction):
         await interaction.response.defer(thinking=False)
         self.LobbyPrompt.clear_items()
+        lobby_details = LobbyPrompt.get_lobby_details(interaction.user)
+
+        if lobby_details != {}:
+            await self.LobbyPrompt.delete_prompt()
+            return await interaction.followup.send(
+                embed=interaction.client.create_embed(
+                    "MOCBOT LOBBIES",
+                    "You have already created a lobby in this server.",

```

Comment:
Error message may need to be more generic. There is a case that exists where the user could have joined a lobby as a member